packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -84,9 +100,21 @@ class _MyHomePageState extends State<MyHomePage> {
         title: new Text(config.title),
       ),
       body: new Center(
+{{#withPluginHook}}
+        child: new Column(
```

Comment:
I'm not sure what to do about the 'multiple versions' part. We deliberately did the conditional approach, as the alternative would have been to fork ALL ~60 files in /templates/create/ into /templates/plugin/ which would have been a lot less scalable.

Do you have a better idea than the conditional approach?